tabular/setup.py

```diff
@@ -34,7 +34,7 @@
 
 extras_require = {
     "lightgbm": [
-        "lightgbm>=4.0,<4.6",  # <{N+1} upper cap, where N is the latest released minor version
+        "lightgbm>=4.6,<4.7",  # <{N+1} upper cap, where N is the latest released minor version
```

Comment:
Why so strict? Why not `>=4.0,<4.7`?

Reply:
The minimum version of LightGBM to mitigate the security vulnerability is 4.6.0, as outlined in this [report](https://nvd.nist.gov/vuln/detail/CVE-2024-43598).